server/home/home-web/src/main/java/io/holoinsight/server/home/web/filter/Step3AuthFilter.java

```diff
@@ -79,7 +79,7 @@ public boolean auth(HttpServletRequest req, HttpServletResponse resp) throws Thr
       String token = req.getHeader("apiToken");
       // 接口权限判定
       if (!ulaFacade.authFunc(req) && StringUtil.isBlank(token)) {
-        log.error("no function authed");
+        log.warn("authFunc check failed");
```

Comment:
Add more detail info about this warn.
Just "authFunc check failed" is not enough to figure out which user/tenant is going wrong.